tool/tsh/tsh.go

```diff
@@ -3786,10 +3786,15 @@ func onShow(cf *CLIConf) error {
 // printStatus prints the status of the profile.
 func printStatus(debug bool, p *profileInfo, env map[string]string, isActive bool) {
 	var prefix string
-	duration := time.Until(p.ValidUntil)
 	humanDuration := "EXPIRED"
+	duration := time.Until(p.ValidUntil)
 	if duration.Nanoseconds() > 0 {
-		humanDuration = fmt.Sprintf("valid for %v", duration.Round(time.Minute))
+		// Display <1m if less then 1 minute left while valid instead of 0s
+		if duration < time.Minute {
+			humanDuration = "valid for <1m"
+		} else {
+			humanDuration = fmt.Sprintf("valid for %v", duration.Round(time.Minute))
+		}
```

Comment:
```suggestion
		humanDuration = fmt.Sprintf("valid for %v", duration.Round(time.Minute))
		// If certificate is valid for less than a minute, display "<1m" instead of "0s".
		if duration < time.Minute {
			humanDuration = "valid for <1m"
		}
```